specreduce/utils/utils.py

```diff
@@ -0,0 +1,170 @@
+import copy
+
+import numpy as np
+
+from specreduce.tracing import Trace, FlatTrace
+from specreduce.extract import _ap_weight_image
+
+
+def measure_cross_dispersion_profile(image, trace=None, crossdisp_axis=0,
+                                     width=None, pixel=None, pixel_range=None,
+                                     statistic='median'):
+    """
+    Return a 1D (quantity) array of the cross-dispersion profile measured at a
+    specified pixel value ('wavelength', but potentially before calibration),
+    or the average profile across several pixel values (or range of pixel values)
+    along the dispersion axis.
+
+    If a single number is specified for `pixel`, then the profile at that pixel
+    (i.e wavelength) will be returned. If several pixels are specified in a list or
```

Comment:
If pixels are mapped to wavelength, can user input the wavelength directly? Does it need units?

Reply:
my take is that the core routine for this should work in pixel space. if we want to work in wavelength space, then we can wrap or decorate this to accept a WCS and input wavelengths to determine `pixel` and `pixel_range`.

Reply:
are you ok with that being a follow up effort?